plugin/pkg/scheduler/factory/factory.go

```diff
@@ -990,3 +991,25 @@ func (p *podConditionUpdater) Update(pod *v1.Pod, condition *v1.PodCondition) er
 	}
 	return nil
 }
+
+type podPreemptor struct {
+	Client clientset.Interface
+}
+
+func (p *podPreemptor) PreemptPod(pod *v1.Pod) error {
+	deleteOptions := &metav1.DeleteOptions{}
+	deleteOptions.GracePeriodSeconds = pod.Spec.TerminationGracePeriodSeconds
```

Comment:
I am not sure this line is necessary. Please check with someone from API machinery. Existing PodSpec terminationGracePeriodSeconds may be used by default.

Reply:
You are right. I just checked with @caesarxuchao and removed the line.